### PR TITLE
DevEx: Whole lotta package updates, November 2023 edition

### LINKED ISSUE #502: [DevEx]: upgrade Vuetify to 3.4.0
### Improvement Summary

Cuttle currently uses a v-skeleton-loader in the StatsView page to manage loading state. This is an experimental Vuetify component with an unstable api that will be finalized in Vuetify’s 3.4.0 (Blackguard) release. We should upgrade to 3.4.0 when it becomes available and remove the Vuetify labs imports and references as we switch to the stable version of v-skeleton-loader

### Detailed Description

As of #500 , Cuttle uses the not-quite-stable `v-skeleton-loader` component from Vuetify labs. 

We should replace this with the stable version and remove the Vuetify lab’s reference by upgrading to Vuetify 3.4.0 once it’s available. As seen in #500 , this should include updating `src/plugins/vuetify.js` to remove the import of  `v-skeleton-loader` as it will be included as a stable component with the rest of the imports (instead of being imported separately from Vuetify labs).

### PR DESCRIPTION
**Pull request by seriouslysean**
## Issue number

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #502 

## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

### CODE REVIEW

**Comment by seriouslysean** [comment]
@itsalaidbacklife @Haviles04 full disclosure, I haven't run the app yet. Just had it in mind to update deps today and wanted to see how many needed it. 👍🏼

File: src/components/MarkdownContent.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@ const text = computed(() => parse(props.markdown, {
 </script>
 
 <template>
+  <!-- eslint-disable-next-line vue/no-v-html -->
```
Comment (itsalaidbacklife) [inline src/components/MarkdownContent.vue:19]: lol

Comment (seriouslysean) [reply src/components/MarkdownContent.vue:19]: I know it's evil, let it happen :)

File: src/components/TheHeader.vue [hunk 2]
```diff
@@ -14,7 +14,7 @@
           >
         </div>
       </v-toolbar-title>
-      <v-toolbar-items v-if="!smAndDown" class="hidden-xs-only">
+      <v-toolbar-items v-if="!smAndDown" class="hidden-xs">
```
Comment (itsalaidbacklife) [inline src/components/TheHeader.vue:17]: What's up with this change? Did a vuetify update change this? Did we have it wrong before?

File: src/components/TheLanguageSelector.vue [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <template>
   <v-menu location="end">
-    <template #activator="{ props }">
+    <template #activator="{ props: activatorProps }">
```
Comment (seriouslysean) [inline src/components/TheLanguageSelector.vue:3]: Aliased this to fix the eslint warning.

File: src/routes/stats/components/StatsLeaderboardCell.vue [hunk 4]
```diff
@@ -9,7 +9,6 @@
           ...props,
           ...dataAttribute,
         }"
-        dark
```
Comment (seriouslysean) [inline src/routes/stats/components/StatsLeaderboardCell.vue:12]: This is no longer supported and I don't know of another prop with a similar result. 🤷🏼

**Comment by Haviles04** [comment]
> @itsalaidbacklife @Haviles04 full disclosure, I haven't run the app yet. Just had it in mind to update deps today and wanted to see how many needed it. 👍🏼

No need for NPM, when you have SPM

File: src/routes/stats/components/StatsLeaderboardCell.vue (hunk 4 above)
Comment (Haviles04) [reply src/routes/stats/components/StatsLeaderboardCell.vue:12]: I'm not sure this actually matters since we don't have a dark/light theme

**Review by Haviles04 — COMMENTED** [review]
I like it, Do we need to do anything to the skeleton loaders now that it's part of vuetify or do they work as-is?

**Comment by seriouslysean** [comment]
@Haviles04 i don’t think so, the API didn’t change. It’s just the full release.

**Review by Haviles04 — APPROVED** [review]

**Pull request merged**